int32 self_id
int32 target_id
---
bool success
